packages/dds/merge-tree/src/localReference.ts

```diff
@@ -125,7 +132,7 @@ class LocalReference implements LocalReferencePosition {
 		this.offset = offset;
 	}
 
-	public isLeaf(): boolean {
+	public isLeaf(): this is ISegment {
```

Comment:
vs code was going crazy with errors without this, but the cmd line build was fine. the change is safe, just odd it had to be made